requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
I've told you: there are no stupid questions! ;-)

Yes these dependencies are not needed for TACA to work, but if you're developing and you want to run the tests locally, you need `nose`. If you want to `make` the docs before pushing changes to GitHub, then you will need `sphing`. Finally if you want you local docs to look exactly like [this](http://taca.readthedocs.org/en/latest/) you will need `sphinx-rtd-theme`.